src/pages/docs/step-cli/the-step-command.mdx

```diff
@@ -90,6 +89,13 @@ There is no support for automatically migrating a single-profile `$STEPPATH` con
 Enabling contexts will preserve any previously existing configuration or state data in `$STEPPATH`,
 but it will not migrate your existing configuration tree into its own new context.
 
+## Extending `step`
+
+The `step` command supports plugins for adding command groups.
+A plugin is an executable file called `step-COMMANDNAME` located in `$PATH` or `$STEPPATH/plugins`.
+For example, a plugin named `$STEPPATH/plugins/step-dns` will execute when `step dns` is run.
+If you're writing a plugin, please follow our convention for CLI help by implementing eg. `step dns help`.
```

Comment:
@maraino occurred to me as I wrote this that we support `step plugin help` but not `step help plugin`

Reply:
Can you add an issue?